src/main/scala/dpla/ingestion3/harvesters/Harvester.scala

```diff
@@ -1,38 +1,21 @@
 package dpla.ingestion3.harvesters
 
-import org.apache.commons.codec.digest.DigestUtils
-
 import scala.xml.Node
 
 /**
   * Base harvester
   */
 object Harvester {
+  // TODO: To ensure continuity of IDs between ingestion sytsems we need to
+  // TODO: parameterize the path to the ID.
   /**
-    * The only shared method between harvester implementations. Generates
-    * an md5 hash of the id value
-    *
-    * @param id String
-    * @param prov String
-    *             Optional, Provider abbreviation if provided is prepended to
-    *             id and then hashed
-    * @return String md5 hash of the id value
-    */
-  def generateMd5(id: String, prov: String=""): String = {
-    if(prov.nonEmpty)
-      DigestUtils.md5Hex(List(prov,id).mkString("--").trim)
-    DigestUtils.md5Hex(id)
-  }
-
-  /**
-    * Accepts a record from an OAI feed an returns the local identifier
-    * TODO: parameterize the path to the ID
+    * Accepts a record from an OAI feed an returns the OAI identifier
     *
     * @param record String
     *               The original record from the OAI feed
     * @return The local identifier
     */
-  def getLocalId(record: Node): String = {
+  def getOaiIdentifier(record: Node): String = {
     (record \\ "header" \\ "identifier").text
   }
```

Comment:
Could the above method be moved to `OaiResponseProcessor`?  There is [similar logic](https://github.com/dpla/ingestion3/blob/964d4db7c6a635b371cf127cb0d21ea6db777e38/src/main/scala/dpla/ingestion3/harvesters/oai/OaiResponseProcessor.scala#L54) in `OaiResponseProcessor` already for getting a set ID, so it would make sense that the logic for getting a record ID follow the same pattern.  Also, I believe that this way we wouldn't need the `Harvester` class at all.